packages/salesforce-adapter/src/filters/custom_objects_instances.ts

```diff
@@ -561,10 +565,21 @@ const createInvalidAliasFieldFetchWarning = async (
 
 const createInvalidManagedBySaltoFieldFetchWarning = async (
   { objectType, invalidManagedBySaltoField }: CustomObjectFetchSetting
```

Comment:
L569 comment hinted on a code smell that can be avoided by enforcing it on the received type.
This can be done in many ways, for example: 
```suggestion
const createInvalidManagedBySaltoFieldFetchWarning = async (
  { objectType, invalidManagedBySaltoField }: CustomObjectFetchSettings & Required<Pick<CustomObjectFetchSettings, 'invalidManagedBySaltoField'>>
```